app/requirements.txt

```diff
@@ -1,6 +1,7 @@
 -e git+https://github.com/linsomniac/python-memcached.git@37f55ca4ad94ca4ade30d6be28e1facb79ac3182#egg=python-memcached
```

Comment:
Not part of this PR, but I'm curious why we have to use the github python-memcached URL instead of the pip package? Is there some funky open bug?